### DISCUSSION TITLE
Error when loading the script

### DISCUSSION THREAD

**Discussion by SupremmeNig** [psvks/NotifyControl#9] [General]
It says invalid URL.  It will be helpful if you guys help. thanks

**Comment by psvks** [comment]
You should check [udkpg's snub api](https://github.com/udkpg/snub-api)

Check if your exploit is supported and then reply back.
snub is inside of NotifyControl for optimizing the script.

Thanks. Remeber to reply back.

**Comment by NotifyControl** [comment]
Hey there, this is an automatic message from Notify Control

This issue is very common when used [udkpg's snub api](https://github.com/udkpg/snub-api)

To fix this issue you may follow these steps:

* Exploit unsupported
   - your exploit may be unsupported by snub api. Recomended exploits are: **comet, synapse, krnl, scriptware and sentinel**
   - download any of these now.
 * Report to snub api's team
   - Report the problem if this issue resists.
 * Use the recent version of notifyControl
   - make sure you're using the latest update of notifyControl
   - if not use the loadstring for autoupdates

***If the problem still hapening you may check the latest console log that snub api has. Thanks, reply back!***

**Comment by SupremmeNig** [comment]
Thank you for reply. 

![image](https://user-images.githubusercontent.com/113242868/190184068-eba94ac0-1f63-402e-9a16-227b44018a0f.png)
This is what appears on the link.  
This may be contacted by the snub api team. The log just updated now,

**Comment by NotifyControl** [comment]
Hi there. Thanks for replying.

***this is an automatic message from Notify Control***

Thanks for the attachment. The problem has been reported to the SNUB's team.  
this might be an error by snub api.
